.ado/jobs/integration-test.yml

```diff
@@ -10,12 +10,11 @@ parameters:
     default:
       - BuildEnvironment: PullRequest
         Matrix:
-          # See #9015
-          # - Name: X64ReleaseHermes
```

Comment:
The commented out bits were supposed to be X64WebDebug. That was the failing configuration...